bazel_tools/scalapb.bzl

```diff
@@ -1,5 +1,5 @@
 # Copyright (c) 2022 Digital Asset (Switzerland) GmbH and/or its affiliates. All rights reserved.
 # SPDX-License-Identifier: Apache-2.0
 
-scalapb_version = "0.11.6"
-scalapb_protoc_version = "0.9.3"
+scalapb_version = "0.11.8"
+scalapb_protoc_version = "0.9.5"
```

Comment:
Perhaps move these alongside the other versions, so they're easier to synchronize.

Reply:
i think these were intentionally broken out since scalapb is used in bazel_tools themselves, so the versions cannot be managed in the jvm-dependencies only file